### PR TITLE
Added an extra step to nuke deconstruction.

### PR DESCRIPTION
**Pull request by xxalpha**
As requested. Use wrench to bang it open.

E: Made it possible to stuff radiation suits in backpacks.

### CODE REVIEW

**Comment by Cheridan** [comment]
People already complain that the steps to dismantling the thing are too obtuse, is this really necessary?

**Comment by xxalpha** [comment]
>As requested

Well yes, people are complaining.

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 1]
```diff
@@ -109,10 +110,28 @@ var/bomb_set
...
-						deconstruction_state = NUKESTATE_CORE_EXPOSED
+						user << "<span class='notice'>You cut into [src]'s warhead. The last containment plate seems to be stuck...</span>"
+						deconstruction_state = NUKESTATE_BREACHED
 						update_icon()
-						SSobj.processing += core
+				return
+		if(NUKESTATE_BREACHED)
+			if(istype(I, /obj/item/weapon/wrench))
+				playsound(loc, 'sound/effects/bang.ogg', 100, 1)
+				user << "<span class='notice'>You hit the containment plate with [I].</span>"
```
Comment (phil235) [inline code/game/gamemodes/nuclear/nuclearbomb.dm]: With this message some players will assume that it's the standard hit message and that the wrench doesn't do anything. A verb other than "hit" would be nice.

**Comment by xxalpha** [comment]
bang :bangbang:

**Comment by WJohn** [comment]
I'm really not a fan of nuke deconstruction. It's absolutely ridiculously terribly handled. The pen is NEVER used in any other deconstruction, for example. There's a custom container just for holding the nuke core that only traitors can get etc.

**Pull request closed without merging**